android/app/src/test/java/org/smartregister/fhircore/activity/PatientListActivityTest.kt

```diff
@@ -63,6 +67,50 @@ class PatientListActivityTest : ActivityRobolectricTest() {
     Assert.assertNotNull(patientListActivity)
   }
 
+  @Test
+  fun testVerifyPatientSearchEditTextDrawables() {
+    val editText = patientListActivity.findViewById<EditText>(R.id.edit_text_search)
+
+    Assert.assertNotNull(editText)
+    Assert.assertNull(editText.compoundDrawables[0])
+
+    editText.setText("")
+    Assert.assertNotNull(editText.compoundDrawables[0])
+
+    editText.setText("demo")
+    Assert.assertNull(editText.compoundDrawables[0])
+    Assert.assertNotNull(editText.compoundDrawables[2])
```

Comment:
how do we know to look at index 2?

Reply:
Basically, the order of drawables are fixed. https://github.com/opensrp/fhircore/blob/dfa5467a68d99edf76a6be25ab63b7f361db7530/android/app/src/main/java/org/smartregister/fhircore/activity/PatientListActivity.kt#L63-L68